Taskfile.yml

```diff
@@ -117,6 +117,7 @@ tasks:
 
   fix:all:
     desc: "Run all fixes"
+    ignore_error: true
```

Comment:
why this?

Reply:
An example:
If the first task in the sequence finds anything to fix, the result ("An error", even though it just outputs that it has fixed x amount of errors) causes the task to stop. ignore_errors makes it execute all the tasks in the sequence, no matter the result of each.